openshift4/templates/backend.dc.yaml

```diff
@@ -315,6 +315,16 @@ objects:
                     secretKeyRef:
                       key: common-services-email-host
                       name: common-services
+                - name: EMAIL_ENABLED
+                  valueFrom:
+                    secretKeyRef:
+                      key: email-enabled
+                      name: common-services
+                - name: EMAIL_RECIPIENT_OVERRIDE
```

Comment:
Is this meant to be empty? Or is an override always provided?

Reply:
If you want to override all email recipients, then it should not be empty. If it is empty then it will not override the email recipients.